tests/bears/BearTest.py

```diff
@@ -262,13 +269,44 @@ def test_new_result(self):
         expected = Result.from_values(bear, 'test message', '/tmp/testy')
         self.assertEqual(result, expected)
 
+
+class BearDownloadTest(BearTestBase):
+
+    def setUp(self):
+        super().setUp()
+        self.mock_url = 'https://test.com'
+        self.filename = 'test.html'
```

Comment:
Okay, just realized that `requests_mock.Mocker()` is a loading mechanism to ensure the adapter is correctly in place to intercept calls from requests. Ignore ^^ above comment.